camptix.php

```diff
@@ -470,9 +470,18 @@ function enqueue_scripts() {
 			'enterEmail' => __( 'Please enter the e-mail addresses in the forms above.', 'camptix' ),
 			'ajaxURL'    => admin_url( 'admin-ajax.php' ),
 		) );
-
 		// Let's play by the rules and print this in the <head> section.
 		wp_enqueue_style( 'camptix' );
+
+
+		if ( (bool) $this->options['select2_enabled'] ) {
```

Comment:
decisions not options?

Reply:
I want a configuration because of accessibility (https://github.com/select2/select2/issues/3744) . Looks like this is also the blocker for merging select2 in core. What do you think ? At the very least, we should enable by default.

Reply:
That's a good point. They could always call `wp_dequeue_script()` from a plugin to disable it, but I realize that'd require some development experience.

It sounds like one our coworkers has already [forked Select2 to make it accessible](https://github.com/woocommerce/selectwoo), so maybe we should switch to that instead?

If you feel like it, it might even be good to send a PR to selectWoo to merge in the upstream changes. That'd help us by fixing any bugs that haven't made it downstream; help selectWoo users for the same reason; and eventually help WP Core and Drupal, by making it easy to merge the a11y fixes upstream.

In general we try to be good open source citizens, but of course it always has to be balanced against time spent on our own priorities. This seems like a really good opportunity, though.

Reply:
I have send in some PRs in wooSelect (https://github.com/woocommerce/selectWoo/pull/18 and https://github.com/woocommerce/selectWoo/pull/17) but it will take a while before they get merged. I want to wait till they get before we use wooSelect, since it doesn't have downstream changes, but we can use it right away as well. What do you think ?

Reply:
It sounds like they'll be looking at it soon, so waiting is probably fine in this case. If it takes them longer than expected, then we can always go ahead and use the setup the latest `master` build now, and then update it after your PRs are merged.